backend/models/user.js

```diff
@@ -127,6 +127,22 @@ const userSchema = new mongoose.Schema(
         ref: "Problem",
       },
     ],
+    weights: {
+      type: [Number],
+      default: [2.8152907722816307, -0.16979919111602879, 0.08516121149998879],
+      validate: {
+        validator: function (v) {
+          return v.length === 3;
+        },
+        message: (props) => `${props.value} must have exactly 3 elements!`,
+      },
+    },
+    bias: {
```

Comment:
FYI: when doing a get from the FE this will always run as long as we dont have any bias values, so the bias values will always be random given that we haven't put value in as the bias